src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -535,11 +535,20 @@ def _create_pgm_input_transformers(self):
         parallel = self._get_pp_attr("trafo", "parallel", 1)
         sn_mva = self._get_pp_attr("trafo", "sn_mva")
         switch_states = self.get_switch_states("trafo")
-        winding_types = self.get_trafo_winding_types()
 
-        tap_side = self._get_pp_attr("trafo", "tap_side")
+        tap_side = self._get_pp_attr("trafo", "tap_side", np.nan)
         tap_nom = self._get_pp_attr("trafo", "tap_neutral", np.nan)
         tap_pos = self._get_pp_attr("trafo", "tap_pos", np.nan)
+        # Do not use taps when mandatory tap data is not available
+        no_taps = np.equal(tap_side, None) + np.isnan(tap_pos) + np.isnan(tap_nom)
```

Comment:
Use plus is confusing here. Consider use element wise or operator `|`. Or even use the function `np.logical_or`.